_studio/mfx_lib/shared/include/mfx_h264_encode_vaapi.h

```diff
@@ -229,6 +229,7 @@ namespace MfxHwH264Encode
         VABufferID m_rirId;                     // VAEncMiscParameterRIR
         VABufferID m_qualityParamsId;           // VAEncMiscParameterEncQuality
         VABufferID m_miscParameterSkipBufferId; // VAEncMiscParameterSkipFrame
+        VABufferID m_maxSliceSizeId;            // VAEncMiscParameterMaxSliceSize
```

Comment:
need to init at constructor

Reply:
@dmitryermilov , please explain what is wrong with mfx_h264_encode_vaapi.cpp, line 1329:

>     , m_maxSliceSizeId(VA_INVALID_ID)